adbpyg_adapter/adapter.py

```diff
@@ -4,9 +4,14 @@
 from collections import defaultdict
 from typing import Any, DefaultDict, Dict, List, Set, Union
 
+try:
+    # https://github.com/arangoml/pyg-adapter/issues/4
+    from cudf import DataFrame
+except ModuleNotFoundError:
+    from pandas import DataFrame
```

Comment:
^ #4

Reply:
reverted [847ebae](https://github.com/arangoml/pyg-adapter/pull/11/commits/847ebae9fd904d5b7d0924a23929ea8b193d6557) in due to https://github.com/arangoml/pyg-adapter/runs/7644978211?check_suite_focus=true (will need to further investigate)